scripts/yarn/downloadReactIsForPrettyFormat.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const path = require('path');
+const pacote = require('pacote');
+const rimraf = require('rimraf');
+const prettyFormatPkg = require('pretty-format/package.json');
+const prettyFormatPkgPath = require.resolve('pretty-format/package.json');
+
+const reactIsDependencyVersion = prettyFormatPkg.dependencies['react-is'];
+
+if (!reactIsDependencyVersion) {
+  throw new Error('Unable to find `react-is` dependency in `pretty-format`');
+}
+
+const prettyFormatNodeModulesReactIsDir = path.join(
+  path.dirname(prettyFormatPkgPath),
+  'node_modules/react-is'
+);
+
+rimraf.sync(prettyFormatNodeModulesReactIsDir);
+
+pacote
```

Comment:
could possibly check in the tarball to avoid the network hit? Not sure if you want this at all, so I went with what's easiest/quickest/shortest